pkg/roachprod/install/cluster_synced.go

```diff
@@ -894,7 +895,7 @@ dead_parent() {
 {{ range .Processes }}
 monitor_process_{{$.Node}}_{{.Name}}_{{.Instance}}() {
   {{ if $.IgnoreEmpty }}
-  if ! ls {{storeFor .Name .Instance}}/marker.* 1> /dev/null 2>&1; then
+  if ! find {{storeFor .Name .Instance}} -name 'marker.*' 2> /dev/null | grep . > /dev/null; then
```

Comment:
We won't actually see `marker.*` under `InstanceStoreDir`, right?

Reply:
I'll have to understand why we search for `marker` as well, and if we should even be doing an IgnoreEmpty check for external process tenants?